src/app.js

```diff
@@ -3,18 +3,26 @@
 const _ = require('lodash');
 const prompt = require('prompt');
 const colors = require('colors');
-const kcomb = require('foreach-combination');
 
 // Get essential data
 const monsters = require('./lib/monsters.js');
 const bosses = require('./lib/bosses.js');
 const specials = require('./lib/specials.js');
 const friends = require('./lib/friends.js');
 const debug = require('./debug.js');
+const mapEngine = require('./map-engine.js');
 const config = require('./config.js');
 const battleEngine = require('./battle-engine.js');
 const player = require('./lib/player.js');
 
+// Map properties
+var map = mapEngine.map;
+
+
+var map_width = config.map_width;
+var map_height = config.map_height;
+var map = create2dArray(map_width,map_height,0);
+
```

Comment:
All looks good, it seems github just inserts the changes rather than updating them. I will remove them myself after merge. Thanks.